core/server/master/src/main/java/alluxio/master/backup/BackupLeaderRole.java

```diff
@@ -183,6 +185,7 @@ public BackupStatus backup(BackupPRequest request, StateLockOptions stateLockOpt
       // Store backup id to query later for async requests.
       backupId = mBackupTracker.getCurrentStatus().getBackupId();
     }
+    AlluxioEvent.BackupStarted.fire(backupId);
```

Comment:
should this have a label for the id, like `String.format("backupId: %d", backupId)`